tools/webpack/shared.js

```diff
@@ -25,7 +25,7 @@ const baseConfig = {
 				parallel: true,
 				terserOptions: {
 					output: {
-						comments: /translators:/i,
+						comments: /(translators|wp):/i,
```

Comment:
That's cool how it can get preserved with little effort.

Reply:
Apparently a little more effort is needed, as discussed below 😄

I'll look for `wp:polyfill` explicitly, instead of just `wp:`

Reply:
Done.